hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusGetTopicInfoHandler.java

```diff
@@ -165,6 +165,9 @@ private Optional<ConsensusTopicInfo> infoForTopic(
             if (!isEmpty(meta.submitKey())) info.submitKey(meta.submitKey());
             info.autoRenewPeriod(Duration.newBuilder().seconds(meta.autoRenewPeriod()));
             if (meta.hasAutoRenewAccountId()) info.autoRenewAccount(meta.autoRenewAccountId());
+            if (meta.hasFeeScheduleKey()) info.feeScheduleKey(meta.feeScheduleKey());
```

Comment:
Should we also check if the key is not empty here? Similar to other key checks

Reply:
Done